iris_grib/_save_rules.py

```diff
@@ -634,17 +633,63 @@ def grid_definition_template_30(cube, grib):
                      int(np.round(first_x / _DEFAULT_DEGREES_UNITS)))
     gribapi.grib_set(grib, "LaD", cs.central_lat / _DEFAULT_DEGREES_UNITS)
     gribapi.grib_set(grib, "LoV", central_lon / _DEFAULT_DEGREES_UNITS)
+
+
+def grid_definition_template_20(cube, grib):
+    """
+    Set keys within the provided GRIB message based on
+    Grid Definition Template 3.20.
+
+    Template 3.20 is used to represent a Polar Stereographic grid.
+
+    """
+    gribapi.grib_set(grib, "gridDefinitionTemplateNumber", 20)
+
+    # Retrieve the cube coord system.
+    cs = cube.coord(dimensions=[0]).coord_system
+
+    _perspective_projection_common(cube, grib)
+
+    # Is this a north or south polar stereographic projection?
+    if cs.central_lat == -90.0:
+        # XXX: is this the correct value? We read bit 0x80 in load, but set bit
+        #      0x1 when saving GDT30...
```

Comment:
I don't know, is it the correct value? This is a very unnerving comment.

Reply:
> This is a very unnerving comment

Yup, it is. I assumed that it should be `0x80` but it was a bit odd that the GDT30 saver did not set that value...

Reply:
> it was ***a bit odd*** that the GDT30 saver did not set that value

Bugs in existing implementation -- doesn't sound that unusual to me :rofl: